src/org/ods/service/OpenShiftService.groovy

```diff
@@ -300,13 +300,38 @@ class OpenShiftService {
       def pod = [ : ]
         pod.podName = podOCData?.metadata?.name?: "N/A"
         pod.podNamespace = podOCData?.metadata?.namespace?: "N/A"
-        pod.podCreationTimestamp = podOCData?.metadata?.creationTimestamp?: "N/A"
-        pod.podEnvironment = podOCData?.metadata?.labels?.env?: "N/A"
+        pod.podMetaDataCreationTimestamp = podOCData?.metadata?.creationTimestamp?: "N/A"
+        pod.deploymentConfigCreationEnvironment = podOCData?.metadata?.labels?.env?: "N/A"
```

Comment:
Just as a side note - I realised we do not set this label anymore, and I would suggest to remove reading it here.

Reply:
removed .env